python/kserve/pyproject.toml

```diff
@@ -53,6 +53,7 @@ timing-asgi = "^0.3.0"
 tabulate = "^0.9.0"
 pandas = ">=1.3.5"
 pydantic = ">1.0,<3"
+openai = "^1.13.3"
```

Comment:
After thought, I think we still should make openai optional as there are model servers that do not implement openai endpoints